src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -241,7 +244,7 @@ bool emberAfOperationalCredentialsClusterSetFabricCallback(chip::app::Command *
 
     // Store vendorId
     admin->SetVendorId(VendorId);
-    emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: vendorId is now set %d", admin->GetVendorId());
+    emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: vendorId is now set %" PRIX16, admin->GetVendorId());
```

Comment:
Below you print vendor ID as a decimal string. Could we make it consistent and use PRId16 or PRIx16 everywhere? In case you choose PRIx16 please add "0x" prefixes.

Reply:
agreed - corrected this to hex for consistency in the rest of the file but missed the 0x